packages/use-intl/src/core/createFormatter.tsx

```diff
@@ -158,6 +158,40 @@ export default function createFormatter({
     );
   }
 
+  function dateTimeRange(
```

Comment:
Would you like to add docs for this too? See https://github.com/amannn/next-intl/blob/main/docs/pages/docs/usage/dates-times.mdx

Reply:
yess